fuzz/fuzz_targets/fuzz_service.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2020, Arm Limited, All Rights Reserved
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//          http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+#![no_main]
+
+use parsec::utils::{ServiceBuilder, ServiceConfig};
+use parsec::authenticators::ApplicationName;
+use parsec::fuzz_utils::requests::Request;
```

Comment:
I don't think these two `use` are needed anymore. cf the comment on the `parsec-interface-rs` `fuzz` feature :)